scripts/source-maintenance.sh

```diff
@@ -119,13 +208,23 @@ if test "${ASVER}" != "${TargetAstyleVersion}" ; then
 		echo "Formatting anyway, please double check output before submitting"
 	fi
 else
-	echo "Found astyle ${ASVER}. Formatting..."
+	echo "Detected expected astyle version: ${ASVER}"
 fi
 CppFormatter=''
-if test "${ASVER}"; then
+if test "${AstyleVersion}"; then
```

Comment:
Something went wrong here. AFAICT, there is no `$AstyleVersion`. Perhaps this change is not needed?

```suggestion
if test "${ASVER}"; then
```